.azure-pipelines-templates/matrix.yml

```diff
@@ -25,7 +25,7 @@ parameters:
     perf:
       cmake_args: '-DBUILD_UNIT_TESTS=OFF -DBUILD_TPCC=ON -DDISTRIBUTE_PERF_TESTS="-n local://localhost -n local://localhost"'
     release:
-      cmake_args: "-DTLS_TEST=ON -DLONG_TESTS=ON -DENABLE_BFT=OFF"
+      cmake_args: "-DTLS_TEST=ON -DLONG_TESTS=ON -DENABLE_BFT=OFF -DENABLE_2TX_RECONFIG=OFF"
```

Comment:
@achamayou Is this over-write of the new option in release mode fine here, or should I add `...=ON` to each of the other configurations?

Reply:
This is fine, imo.